bower.json

```diff
@@ -16,15 +16,15 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^2.5.0",
-    "purescript-parsing": "^3.2.1",
-    "purescript-fixed-points": "^3.0.0",
-    "purescript-datetime": "^2.1.0"
+    "purescript-prelude": "^3.0.0",
+    "purescript-parsing": "^4.0.0",
+    "purescript-fixed-points": "git://github.com/safareli/purescript-fixed-points.git#ps11",
```

Comment:
Should wait before https://github.com/slamdata/purescript-fixed-points/pull/7 is merged